Lib/test/test_msvcrt.py

```diff
@@ -115,6 +115,28 @@ def test_heap_min(self):
         except OSError:
             pass
 
+    def test_GetErrorMode(self):
+        msvcrt.GetErrorMode()
+
+    def test_SetErrorMode(self):
+        old = msvcrt.SetErrorMode(0)
+        msvcrt.SetErrorMode(old)
```

Comment:
We can at least assert these documented things:

```suggestion
        old = msvcrt.GetErrorMode()
        self.addCleanup(msvcrt.SetErrorMode, old)

        returned = msvcrt.SetErrorMode(0)
        self.assertIs(type(returned), int)
        self.assertEqual(old, returned)
```

Reply:
This is a good mechanism. I applied it to other tests. Thanks!

I read Microsoft's document and found that `set_error_mode` and `CrtSetReportMode` may return `-1` for a failure, so I also checked the result in these functions' tests.

`CrtSetReportFile` may return a special value defined as a C macro when failed, but currently msvcrt does not export it to Python. I think we should add it, but I think this will be in a separate PR.